services/power.md

```diff
@@ -4,6 +4,69 @@
 
 A power-provider service.
 
+## Power negotiation protocol
+
+The purpose of the power negotiation is to ensure that there is no more than ~500mA
+delivered to the power rail.
+This is realized by limiting the number of enabled power provider services to 1.
+
+Note, that it's also possible to have low-current power providers, which do not
```

Comment:
link to more info about "low-current power providers" ?

Reply:
no link yet; added info